File: 10-DotNet/3-IntroductionPOO/IntroductionPOO/ExercicesPOO/MyEnvironement/Classes/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyEnvironement.Classes
{
    internal class Person
    {
        private int id;
        private string title;
        private string firstname;
        private string lastname;
        private DateTime dateOfBirth;

        public Person()
        {

        }

        public Person(int id, string title, string firstname, string lastname, DateTime dateOfBirth)
        {
            Id = id;
            Title = title;
            Firstname = firstname;
            Lastname = lastname;
            DateOfBirth = dateOfBirth;
        }

        public int Id { get => id; set => id = value; }
        public string Title { get => title; set => title = value; }
        public string Firstname { get => firstname; set => firstname = value; }
        public string Lastname { get => lastname; set => lastname = value; }
        public DateTime DateOfBirth { get => dateOfBirth; set => dateOfBirth = value; }

        public override string ToString()
        {
            return $"\n{Title} {Firstname} {Lastname}";
        }
    }
}

File: 08-Docker/06-Docker-Compose/ApiContact/Back_Api_Node/classes/person.js
export default class Person {
    constructor(id, title, firstname, lastname, dateOfBirth, urlImg = " ", created = new Date(), updated = " ") {
        this.id = id;
        this.title = title;
        this.firstname = firstname;
        this.lastname = lastname;
        this.dateOfBirth = dateOfBirth;
        this.urlImg = urlImg;
        this.created = created;
        this.updated = updated;
    }

    toString() {
        return `${this.id} ${this.title} ${this.lastname} ${this.firstname} - ${this.dateOfBirth}`;
    }
}